docs/pages/server-access/guides/ec2-discovery.mdx

```diff
@@ -0,0 +1,308 @@
+---
+title: Configure Teleport to Automatically Enroll EC2 instances
+description: How to configure Teleport to automatically enroll EC2 instances.
+---
+
+Teleport can run as a Discovery Node by configuring the discovery
+service. This will connect to EC2 and automatically discover and
+enroll EC2 instances matching configured labels. It will then execute
+an install script on these discovered instances using AWS Systems
+Manager that will install Teleport, start it and join the cluster.
+
+## Prerequisites
+
+(!docs/pages/includes/edition-prereqs-tabs.mdx!)
+
+- AWS account with EC2 instances and permissions to create and attach IAM policies as well as permission to create Systems Manager documents.
+- EC2 instances running Ubuntu/Debian/RHEL/Amazon Linux 2 if making use of the default Teleport install script. 
```

Comment:
```suggestion
- EC2 instances running Ubuntu/Debian/RHEL/Amazon Linux 2 if making use of the default Teleport install script. (For other Linux distributions, you can install Teleport manually.)
```
Feel free to adjust this suggestion—I think we need explicit instructions for unlisted distributions so users with those distributions don't get confused here.